scripts/CsvToJson.unit.test.js

```diff
@@ -42,7 +60,7 @@ describe('CsvToJson script OK', () => {
     expect(execSpy).toBeCalledWith(
       `${csvToJsonPath} ${validProgram.sourcefilepath} > ${
         validProgram.outputfilepath
-      }`,
+      } `,
```

Comment:
🗡

Reply:
That space is actually intentional 😁

Reply:
Oh, why tho?

Reply:
This particular unit test assertion is based on what is actually executed by the script in the spawned child process.